it/support/stubs/CommonStubs.scala

```diff
@@ -110,11 +130,11 @@ trait CommonStubs {
           .withStatus(200)
           .withFixedDelay(2000) // For the tests to correctly fail this value must be greater than the ws.timeout.request used by the test.
```

Comment:
I know you didn't write this but the `. withFixedDelay` isn't needed for positive tests and is actually slowing this test down